from src.utils.Base.BaseException import CustomBaseError


class InputError(CustomBaseError):
    pass
